UserDetailsClient/UserDetailsClient.Core/Features/LogOn/B2CAuthenticationService.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Microsoft.Identity.Client;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UserDetailsClient.Core.Features.LogOn
+{
+    public class B2CAuthenticationService : IAuthenticationService
+    {
+        public static IPublicClientApplication PCA = null;
```

Comment:
why all those statics?

Reply:
it was originally static, i did minimal code touches to reorganize outside of App.xaml.cs.

I eliminated static usage though.